pages-functions-cors/functions/index.ts

```diff
@@ -0,0 +1,3 @@
+export const onRequestGet: PagesFunction = async () => {
```

Comment:
I'd suggest moving this to a `public/index.html` file for two reasons:
- it shows people how to use static files too
- `wrangler pages dev public` will be looking for a `public` directory, which won't exist here because you don't have any files (and you can't add an empty dir to git)